estate/models/estate_property.py

```diff
@@ -0,0 +1,31 @@
+from odoo import models, fields
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate Property"
+
+    name = fields.Char(string='Property Name', required=True)
+    description = fields.Text(string='Property Description')
+    postcode = fields.Char(string='Property Postcode')
+    date_availability = fields.Date(string='Availability', copy=False, default=fields.Date.add(fields.Date.today(),
+                                                                                               months=3))
```

Comment:
```suggestion
    date_availability = fields.Date(string='Availability', copy=False, default=fields.Date.add(fields.Date.today(), months=3))
```
I think a single line can contain all attributes. If you think it's too long, then the indentation can be improved in a similar way to the comment I give below about the selection field